src/python/turicreate/data_structures/serialization.py

```diff
@@ -0,0 +1,51 @@
+
+
+# SFrames require a disk-backed file or directory to work with.  This directory 
+# has to be present to allow for serialization or deserialization. 
+__serialization_directory = None 
+
+
+def enable_sframe_serialization(serialization_directory):
+    """
+    Enables pickling of sframes through the use of a user-set directory to 
+    store the objects.  This directory must be present and set through 
+    this method for deserialization to work, but may be a different directory.
+    """
+
+    import os
+
+    global __serialization_directory
+    if serialization_directory is None:
+        __serialization_directory = None
+        return 
+
+    __serialization_directory = os.path.abspath(os.path.expanduser(serialization_directory))
+
+    # Make sure the directory exists.
+    if not os.path.exists(__serialization_directory):
+        raise ValueError("Directory %s does not exist." % __serialization_directory)
```

Comment:
I think we should try to create the directory if it doesn't exist. That matches behavior elsewhere in TuriCreate.